tests/integration/cli/tests/run.rs

```diff
@@ -500,7 +536,11 @@ fn wasi_runner_on_disk() {
         .env("RUST_LOG", &*RUST_LOG)
         .assert();
 
-    assert.success().stdout(contains("Hello, World!"));
+    if cfg!(not(feature = "wamr")) {
+        assert.success().stdout(contains("Hello, World!"));
+    } else {
+        assert.success().stderr(contains("Hello, World!"));
+    }
```

Comment:
Why stderr on wamr but stdout normally? This seems like wamr is doing things that it shouldn't.

I found `wasm_runtime_init_wasi` to be the clulprit. But it seems we are not using it.. right?